### PR TITLE
add coinbase wallet

### PR DESCRIPTION
**Pull request by adamsoffer**
**What does this pull request do? Explain your changes. (required)**
This PR adds Coinbase Wallet support

**Does this pull request close any open issues?**
#579 

**Screenshots (optional):**
Screen Shot 2020-03-03 at 10 25 06 PM

**Checklist:**
- [x] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/explorer-2.0/lib/constants.tsx [hunk 1]
```diff
@@ -22,6 +23,23 @@ export const SUPPORTED_WALLETS = {
...
+    icon: CoinbaseWalletIcon,
+    description: 'Use Coinbase Wallet app on mobile device',
+    href: null,
+    color: '#315CF5',
+  },
+  COINBASE_LINK: {
+    name: 'Open in Coinbase Wallet',
+    icon: CoinbaseWalletIcon,
+    description: 'Open in Coinbase Wallet app.',
+    href: 'https://go.cb-w.com/0T8By93MA4',
```
Comment (kyriediculous) [inline packages/explorer-2.0/lib/constants.tsx:38]: Should this be exposed?

Comment (adamsoffer) [reply packages/explorer-2.0/lib/constants.tsx:38]: Yeah, it's just a deep link that takes users directly to the explorer inside the coinbase wallet dapp browser.

**Review by kyriediculous — APPROVED** [review]

File: packages/explorer-2.0/lib/constants.tsx (hunk 1 above)
Comment (kyriediculous) [reply packages/explorer-2.0/lib/constants.tsx:38]: Okay ! LGTM :shipit: 

those deployment previews are really helpful btw, nice job on those !

Comment (adamsoffer) [reply packages/explorer-2.0/lib/constants.tsx:38]: Awesome thanks!

**Pull request merged**